src/main/java/com/laserfiche/api/client/model/ApiException.java

```diff
@@ -0,0 +1,51 @@
+package com.laserfiche.api.client.model;
```

Comment:
Nothing to change. But a reminder that since the repository client also has this class, maybe we should make sure two classes are identical and the repository client can just use this class instead (it should be easy to do, we could do this before release).